deps/npm/docs/content/using-npm/workspaces.md

```
@@ -88,24 +88,25 @@ npm install abbrev -w a
 
 **Adding a workspace as a dependency of another workspace:**
 
-If you want to add workspace **b** as a dependency of workspace **a**, you can use the workspace protocol in the dependency specifier:
+The same approach works when adding one workspace as a dependency of another.
+If you want to add workspace **b** as a dependency of workspace **a**, run:
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)